rdmo/core/urls/openapi.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.decorators import login_required
+from django.urls import path
+
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
+
+# TODO: maybe generate the schema.yml as part of the build process?
+
+urlpatterns = [
```

Comment:
I think either the 'swagger' view should be the root of those urlpatterns (to emulate the current behavior) or we add a small template with links to the 3 endpoints.